TriblerGUI/widgets/trustpage/js/help_page.js

```diff
@@ -0,0 +1,260 @@
+        self._drawLineSizeSvg();
+        self._drawLineDividerSvg();
+    };
+
+    self.show = function () {
+        document.getElementById("help-button").innerHTML = "Close";
+        document.getElementsByTagName("body")[0].className = "show-help-page";
+        self.visible = true;
+    };
+
+    self.close = function () {
+        document.getElementById("help-button").innerHTML = "Help";
+        document.getElementsByTagName("body")[0].className = "";
+        self.visible = false;
+    };
+
+    self.visible = false;
+    self.toggle = function () {
+        self.visible ? self.close() : self.show();
+    };
+
+    /**
+     * Hides the help page and help page button if the window becomes too small
+     */
+    window.onresize = function () {
+        if (window.innerWidth < self.config.help.page.minWindowWidth) self.close();
+    };
+
+    /**
+     * Get node color based on node balance score [0, 1]
+     * @private
+     */
+    self._nodeColor = d3.scaleLinear()
+        .domain(self.config.node.color.domain)
+        .range(self.config.node.color.range);
+
+    /**
+     * Fill the svg which explains the meaning of the text inside the circle.
+     * @private
+     */
+    self._drawCircleTextSvg = function () {
+
+        var svg = d3.select("#help-circle-text"),
+            containerSvg = self._drawSvgCenterContainer(svg)
+                .attr("x", parseInt(svg.style("width")) / 2);
+
+        containerSvg.append("circle")
+            .attr("r", self.config.node.circle.maxRadius)
+            .attr("fill", self.config.triblerOrange);
+
+        containerSvg
+            .append("text")
+            .attr("dominant-baseline", "central")
+            .attr("text-anchor", "middle")
+            .attr("fill", self.config.node.publicKeyLabel.color)
+            .style("font-family", self.config.node.publicKeyLabel.fontFamily)
+            .style("font-size", self.config.node.publicKeyLabel.fontSize)
+            .style("font-weight", self.config.node.publicKeyLabel.fontWeight)
+            .text(self.config.help.examplePublicKey);
+    };
+
+    /**
+     * Fill the svg which explains the meaning of the circle colors.
+     * @private
+     */
+    self._drawCircleColorSvg = function () {
+        var svg = d3.select("#help-circle-color"),
+            containerSvg = self._drawSvgCenterContainer(svg),
+            nodeRadius = self.config.help.circleColor.nodeRadius,
+            names = self.config.help.circleColor.colorNames,
+            numberOfNodes = self.config.help.numberOfNodes,
+            labelOffset = self.config.help.label.offsetY,
+            width = parseInt(svg.style("width")),
+            spacing = width / (numberOfNodes + 1),
+            labelY = (1 / 2) * nodeRadius + labelOffset + self.config.help.label.fontSize,
+            nodesData = [];
+
+        // Fill array with the data belonging to each node
+        for (var i = 0; i < numberOfNodes; i++) {
+            nodesData.push({
```

Comment:
As we have done in the methods below, simply put the append in this for loop. You can then remove the whole nodesData array. Take a good look at how we have done this below.

Together with the ```drawCircleSizeSvg``` you could move this to a separate ```_drawNode``` method